### ISSUE TITLE
[Feature Request] Option to walk while aiming with the snipers

### ISSUE THREAD

**Issue by Otheucarvalho** [ThirteenAG/GTAIV.EFLC.FusionFix#393]
### What should happen

It would be really cool if there was an option to be able to aim with the sniper as I play some fps I miss at least having an option to activate this.

### Checklist

- [X] Check the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already implemented.
- [X] [Search for other requests](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same feature.

**Comment by hammerfist11** [comment]
I think this is beyond the scope of the mod.

**Comment by Panzermann11** [comment]
> I think this is beyond the scope of the mod.

Exactly. The purpose of this mod is to fix issues in the PC port and add quality-of-life improvements, not arbitarily add new gameplay features.

**Comment by RecklessGlue540** [comment]
I still think this could be considered in the future...
It is a pretty big quality of life improvement in my humble opinion. It's in no way a completely new gameplay feature lol...
